.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,29 @@
+# How To Contribute
+To keep our repositories as uniform as possible, and work together effectively please consider the following:
+
+## Our Git Flow
+We use two primary branches.  `master` and `development`.  `development` is what you'll create pull requests against, and should be deployed to staging environments for testing. Once enough work has gone in to create a  deliverable piece value, and all testing is complete, we should merge `development` into `master`, which will then be deployed to a production environment.  Ideally `master` should always represent what is in production. 
+
```

Comment:
Okay, still not sure how you guys want to do this, I've updated it to match the git-flow discussed here, https://www.atlassian.com/git/tutorials/comparing-workflows/gitflow-workflow  which seems to work with release branches, and focus on continuous deployment from master to live.